obspy/io/xseed/tests/test_parser.py

```diff
@@ -849,8 +849,8 @@ def test_parsing_resp_file_with_mutiple_blockette55(self, testdata):
         """
         Test case for issue #3275. Allow reading mutiple blockette 55.
         """
-        inv_xml = obspy.read_inventory('issue3275.xml')
-        inv_seed = obspy.read_inventory('issue3275.seed')
+        inv_xml = obspy.read_inventory(testdata['issue3275.xml'])
+        inv_seed = obspy.read_inventory(testdata['issue3275.seed'])
```

Comment:
@isbreen this is how to lookup test data within one module through the pytest fixture we provide, just fyi